project/AppDependencies.scala

```diff
@@ -8,39 +8,34 @@ private object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "http-caching-client" % "10.0.0-play-28",
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "7.12.0",
+    "uk.gov.hmrc" %% "http-caching-client-play-30" % "11.2.0",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-30" % "8.4.0",
     "uk.gov.hmrc" %% "tax-year" % "3.0.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc-play-28" % "8.5.0"
+    "uk.gov.hmrc" %% "play-frontend-hmrc-play-30" % "8.5.0"
   )
 
   def testCommon(): Seq[ModuleID] = {
     val scope: String = "test,it"
     Seq(
-      "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0"             % scope,
-      "org.jsoup"              % "jsoup"               % "1.15.3"            % scope,
-      "uk.gov.hmrc" %% "bootstrap-test-play-28" % "7.12.0"  % scope,
-      "com.typesafe.play"      %% "play-test"          % PlayVersion.current % scope,
+      "org.scalatestplus.play" %% "scalatestplus-play" % "7.0.1"             % scope,
+      "org.jsoup"              % "jsoup"               % "1.17.2"            % scope,
+      "uk.gov.hmrc" %% "bootstrap-test-play-30" % "8.4.0"  % scope,
+      "org.playframework"      %% "play-test"          % PlayVersion.current % scope,
       "org.scalacheck"         %% "scalacheck"         % "1.17.0"            % scope,
       "org.scalatestplus" %% "mockito-3-12" % "3.2.10.0" % scope,
-      "com.fasterxml.jackson.module" %% "jackson-module-scala"   % "2.14.1" % scope
+      "com.fasterxml.jackson.module" %% "jackson-module-scala"   % "2.16.1" % scope,
```

Comment:
this can go up to 2.17.0
https://github.com/FasterXML/jackson-module-scala/tags

Reply:
done